### PR TITLE
site: add my article

### PR DESCRIPTION
**Pull request by heiyu4585**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    add my article       |
| 🇨🇳 Chinese |      添加文章     |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: docs/blog/to-be-collaborator.en-US.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+- Submit the updated code to your own `fix-branch`:
+
+```markdown
+git push origin fix-branch
+```
+
+## Submit a Pull Request
+
+You can switch to the branches page on your github code warehouse page, click the fix-branch branch, click the `New pull request` button, add relevant comments and submit. Or switch to the code warehouse of the fix-branch branch, click the `Compare & pull request` button, add relevant comments and submit.
```
Comment (abenhamdine) [inline docs/blog/to-be-collaborator.en-US.md:118]: I think by "warehouse" you mean "repository" no ?

File: docs/blog/to-be-collaborator.en-US.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+- react 16 test failed
+
+![image-20221212091630186](https://user-images.githubusercontent.com/10607168/208017142-c9ee4169-f2d0-4085-bcff-6c859ec54e71.png)
+
+Repair method:
+
+```
+npm run install-react-16
+npm run test componet/XXX
```
Comment (abenhamdine) [inline docs/blog/to-be-collaborator.en-US.md:148]:
```suggestion
npm run test component/XXX
```

File: docs/blog/to-be-collaborator.en-US.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+## Run test cases and specification checks
+
+1. Make sure all tests pass `npm run test`. Tips: You can use `npm test -- --watch TestName` to run specified tests during development.
+2. Run `npm test -- -u` to update [jest snapshot](https://facebook.github.io/jest/docs/en/snapshot-testing.html#snapshot-testing-with-jest) and put These updates are also committed (if any).
+3. Make sure your code passes the lint check `npm run lint`.
+
+## Merge changes
+
+- A common problem is that the remote upstream (ant-design/master) has a new update, which will lead to conflicts when we submit the Pull Request, so we can put the commits of other remote developers and Our commit is merged.
```
Comment (abenhamdine) [inline docs/blog/to-be-collaborator.en-US.md:84]:
```suggestion
- A common problem is that the remote upstream (ant-design/master) has a new update, which will lead to conflicts when we submit the Pull Request, so we can put the commits of other remote developers and our commit is merged.
```

**Review by abenhamdine — CHANGES_REQUESTED** [review]

**Comment by heiyu4585** [comment]
@abenhamdine, thx for your warm heart! Since I'm not native speaker. Your helps a lot!

File: docs/blog/to-be-collaborator.en-US.md (hunk 3 above)
Comment (heiyu4585) [reply docs/blog/to-be-collaborator.en-US.md]: done

File: docs/blog/to-be-collaborator.en-US.md (hunk 2 above)
Comment (heiyu4585) [reply docs/blog/to-be-collaborator.en-US.md]: done

File: docs/blog/to-be-collaborator.en-US.md (hunk 1 above)
Comment (heiyu4585) [reply docs/blog/to-be-collaborator.en-US.md]: done

File: docs/blog/to-be-collaborator.zh-CN.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
+---
+title: 如何成长为 Collaborator
+date: 2022-12-08
+author: heiyu4585
+---
+
+大家好，我是[黑雨](https://github.com/heiyu4585)。关于 Ant Design ，相信大家都非常熟悉，我在为 Ant Design 做贡献之前在工作中很多后台管理系统项目使用过 Ant Design，给我最大的感觉就是好用、好看、简单、稳定。现在 `v5` 版本已经发布，强烈推荐大家试使用，我也有幸开发了 `v5` 版本的 `Tour` 组件和 `App` 组件，以及一些其他维护工作，下面给大家分享一下如何通过给 Ant Design 提 PR 成长为 Collaborator。
```
Comment (miracles1919) [inline docs/blog/to-be-collaborator.zh-CN.md:7]: > 我在为 Ant Design 做贡献之前在工作中很多后台管理系统项目使用过 Ant Design

读起来怪怪的

Comment (heiyu4585) [reply docs/blog/to-be-collaborator.zh-CN.md:7]: 我在工作中很多后台管理系统项目使用过 Ant Design   
这样呢

Comment (miracles1919) [reply docs/blog/to-be-collaborator.zh-CN.md:7]: “我在很多后台管理系统项目使用过 Ant Design”

Comment (heiyu4585) [reply docs/blog/to-be-collaborator.zh-CN.md:7]: done

File: docs/blog/to-be-collaborator.zh-CN.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+## 二. 拉取 Ant Design 代码到本地
+
+### 1. Fork 项目
+
+- 首先需要 fork 项目，进入[项目页面](https://github.com/ant-design/ant-design)，点击右上角的 [Fork 按钮](https://github.com/ant-design/ant-design/fork)
+- 你的 github 帐号中会出现 Ant Design 链接是 https://github.com/heiyu4585/ant-design 这个项目
+- 在本地电脑上项目地址使用以下命令：得到一个 github 文件夹
+
+```markdown
```
Comment (MadCcc) [inline docs/blog/to-be-collaborator.zh-CN.md:29]:
```suggestion
```bash
```
这种都可以改成 bash

File: docs/blog/to-be-collaborator.zh-CN.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+```
+
+注意： `heiyu4585` 改为自己的 github 用户名
+
+### 2. 添加远端分支地址
+
+- 进入 Ant Design 文件夹，添加 Ant Design 的远程地址
+
+```
+cd ant-desgin
```
Comment (MadCcc) [inline docs/blog/to-be-collaborator.zh-CN.md:40]:
```suggestion
cd ant-design
```

Comment (heiyu4585) [reply docs/blog/to-be-collaborator.zh-CN.md]: done

File: docs/blog/to-be-collaborator.zh-CN.md (hunk 5 above)
Comment (heiyu4585) [reply docs/blog/to-be-collaborator.zh-CN.md]: done

**Review by zombieJ — APPROVED** [review]

**Pull request merged**